content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md

```diff
@@ -123,31 +123,30 @@ See section [specifying your own pod selector](#specifying-your-own-pod-selector
 
 ### Parallel Jobs
 
-There are three main types of jobs:
+There are three main types of task suitable to run as a Job:
 
 1. Non-parallel Jobs
-  - normally only one pod is started, unless the pod fails.
-  - job is complete as soon as Pod terminates successfully.
+  - normally, only one Pod is started, unless the Pod fails.
+  - the task is complete as soon as its Pod terminates successfully.
 1. Parallel Jobs with a *fixed completion count*:
   - specify a non-zero positive value for `.spec.completions`.
-  - the job is complete when there is one successful pod for each value in the range 1 to `.spec.completions`.
-  - **not implemented yet:** Each pod passed a different index in the range 1 to `.spec.completions`.
+  - the overall task is complete when there is one successful Pod for each value in the range 1 to `.spec.completions`.
```

Comment:
job

Reply:
I'm keen for the reader to get the idea that there's an overall piece of work represented by the Job. Usings might be coming from other systems where “job” could mean a batch from a larger pool, or a even a single element inside a batch.

I'll see if I can make some wording that to reinforce that, without losing the link to the Job concept in a Kubernetes sense.